crm_construction_architect/oca_dependencies.txt

```diff
@@ -0,0 +1,3 @@
+# List the OCA project dependencies, one per line
+
+construction_architect_base https://github.com/OCA/vertical-construction
```

Comment:
remove this line: vertical-construction is where we are so no need to list it in the project dependencies.